api/drop_chunks.md

```diff
@@ -54,19 +56,46 @@ The `older_than` and `newer_than` parameters can be specified in two ways:
     `SMALLINT` / `INT` / `BIGINT`. The choice of timestamp or integer
     must follow the type of the hypertable's time column.
 
+The `created_before` and `created_after` parameters can be specified in two ways:
+
+*   **interval type:** The cut-off point is computed as `now() -
+    created_before` and similarly `now() - created_after`.  This uses
+    the chunk creation time for the filtering.
+
+*   **timestamp, date, or integer type:** The cut-off point is
+    explicitly given as a `TIMESTAMP` / `TIMESTAMPTZ` / `DATE` or as a
+    `SMALLINT` / `INT` / `BIGINT`. The choice of integer value
+    must follow the type of the hypertable's partitioning column. Otherwise
+    the chunk creation time is used for the filtering.
+
 <Highlight type="warning">
 When using just an interval type, the function assumes that
 you are removing things _in the past_. If you want to remove data
 in the future, for example to delete erroneous entries, use a timestamp.
 </Highlight>
 
-When both arguments are used, the function returns the intersection of the
-resulting two ranges. For example, specifying `newer_than => 4 months` and
-`older_than => 3 months` drops all full chunks that are between 3 and 4 months
-old. Similarly, specifying `newer_than => '2017-01-01'` and
-`older_than => '2017-02-01'` drops all full chunks between '2017-01-01' and
-'2017-02-01'. Specifying parameters that do not result in an overlapping
-intersection between two ranges results in an error.
+When both `older_than` and `newer_than` arguments are used, the
+function returns the intersection of the resulting two ranges. For
+example, specifying `newer_than => 4 months` and `older_than => 3
+months` drops all full chunks that are between 3 and 4 months old.
+Similarly, specifying `newer_than => '2017-01-01'` and `older_than
+=> '2017-02-01'` drops all full chunks between '2017-01-01' and
+'2017-02-01'. Specifying parameters that do not result in an
+overlapping intersection between two ranges results in an error.
+
+When both `created_before` and `created_after` arguments are used, the
+function returns the intersection of the resulting two ranges. For
+example, specifying `created_after`=> 4 months` and `created_before`=> 3
+months` drops all full chunks created between 3 and 4 months from now.
+Similarly, specifying `created_after`=> '2017-01-01'` and `created_before`
+=> '2017-02-01'` drops all full chunks created between '2017-01-01' and
+'2017-02-01'. Specifying parameters that do not result in an
```

Comment:
```suggestion
2017-02-01. Specifying parameters that do not result in an
```

Reply:
single quoted dates are used elsewhere as well. Why this specific change?

Reply:
I suggested this change because inside code the quotes makes sense, but in free-flowing text it is just a date and hence the quotes makes it look like some coding artifact when it is just a date. It's like putting quotes around "May 16, 2016" in free flowing text; yes, it can be used inside code as well, but it is actually just a date.

Reply:
Example, just to illustrate:
> On April 5th, 2063 the first contact with an alien species were made.

vs.

> On April 5th, 2063 the first contact with an alien species were made.